Sources/ArcGISToolkit/Documentation.docc/Resources/Authenticator/AuthenticatorStep3.swift

```diff
@@ -4,20 +4,16 @@ import ArcGIS
 
 @main
 struct AuthenticationApp: App {
-    @ObservedObject var authenticator: Authenticator
-    
-    init() {
-        authenticator = Authenticator()
-        ArcGISEnvironment.authenticationManager.handleChallenges(using: authenticator)
-    }
+    @StateObject private var authenticator = Authenticator()
     
     var body: some SwiftUI.Scene {
         WindowGroup {
-            Group {
-                HomeView()
-            }
```

Comment:
Removes the unneeded Group in response to [this feedback](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/392#discussion_r1295003207)